.github/workflows/test_action_locally.yml

```
@@ -0,0 +1,162 @@
+name: Test the action
```

I will love to have more descriptive name and emoji if possible 🤣 

```suggestion
name: ☕️ E2E Tests
```